test/draftertest.h

```diff
@@ -12,20 +12,25 @@
 #include "SerializeResult.h"
 #include "sosJSON.h"
 
+
+#define TEST_DRAFTER(description, category, name, tag,  wrapper, options, mustBeOk) TEST_CASE(description " " category " " name, "[" tag "][" category "]") { \
+    REQUIRE(FixtureHelper::handleResultJSON(wrapper, "test/fixtures/" category "/" name, options, mustBeOk)); \
+}
```

Comment:
I mean, we should do it in same way for other tests.

There is no reason to have specific macro accessible for other test files, it just filling "global namespace"